src/secondPackageForHomework6/CheckCalculator.java

```diff
@@ -0,0 +1,13 @@
+package secondPackageForHomework6;
+
+import homework6.Calculator;
+
+public class CheckCalculator {
+    public static void main(String[] args) {
+
+        System.out.println("Addition result is: " + Calculator.addition(3, 15));
```

Comment:
Can be used static import

Reply:
Changed to `import static homework6.Calculator.*; `